docs/data/material/customization/how-to-customize/how-to-customize.md

```diff
@@ -119,10 +114,10 @@ You can rely on the following [global class names](/styles/advanced/#with-materi
 | required      | `.Mui-required`     |
 | selected      | `.Mui-selected`     |
 
-> ⚠️ Never style these state classes' names directly:
+> ⚠️ Never style these state classes' names directly. If you do that, it will impact all the components with unclear side-effects. Always target it together with a component.
```

Comment:
Rephrasing to avoid the awkward possessive.

```suggestion
> ⚠️ Never apply styles directly to state class names. This will impact all components with unclear side-effects. Always target a state class together with a component.
```